src/main/java/com/api/dto/RocketDto.java

```diff
@@ -0,0 +1,156 @@
+package com.api.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class RocketDto {
```

Comment:
do you really need all the setters and getters? set only needed Annotations and add private to params

Reply:
I delete some of them